File: src/SOP8/Base/Account/Base.Account.IBLL/Request/CheckLoginSession.cs
namespace Base.Account.IBLL.Request
{
    public class CheckLoginSession
    {
        private int m_nUserNo = -1;
        private string m_strSessionKey = "";


        public int UserNo
        {
            get { return m_nUserNo; }
            set { m_nUserNo = value; }
        }

        public string SessionKey
        {
            get { return m_strSessionKey; }
            set { m_strSessionKey = value; }
        }
    }
}

File: src/SOP8/광양/EgovLogin/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/CheckLoginSession.java
package egovframework.base.account.ibll.request;

public class CheckLoginSession {

    private int userNo = -1;
    private String sessionKey = "";

    public int getUserNo() {
        return userNo;
    }

    public void setUserNo(int userNo) {
        this.userNo = userNo;
    }

    public String getSessionKey() {
        return sessionKey;
    }

    public void setSessionKey(String sessionKey) {
        this.sessionKey = sessionKey;
    }
}